pom.xml

```diff
@@ -92,7 +92,7 @@
     <netty-tcnative.classifier>linux-x86_64</netty-tcnative.classifier>
 
     <!-- General Versions -->
-    <guava.version>16.0.1</guava.version>
+    <guava.version>18.0</guava.version>
```

Comment:
Should this change be included in this PR?

Reply:
Nope